osmosis-1/osmosis-1.assetlist.json

```diff
@@ -10597,6 +10597,60 @@
         "osmosis-info",
         "osmosis-price:uosmo:1087"
       ]
+    },
+    {
+      "description": "The native staking token of the Function X",
+      "denom_units": [
+        {
+          "denom": "ibc/EBBE6553941A1F0111A9163F885F7665417467FB630D68F5D4F15425C1E64FDE",
+          "exponent": 0
+        },
+        {
+          "denom": "FX",
+          "exponent": 0
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/EBBE6553941A1F0111A9163F885F7665417467FB630D68F5D4F15425C1E64FDE",
+      "name": "Function X",
+      "display": "FX",
+      "symbol": "FX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "fxcore",
+            "base_denom": "FX",
+            "channel_id": "channel-10"
```

Comment:
Please register this IBC connection to the Chain Registry (under /_IBC as cosmoshub-fxcore.json)